lib/spi.ex

```diff
@@ -88,6 +88,14 @@ defmodule Circuits.SPI do
   @spec info() :: map()
   defdelegate info(), to: Nif
 
+  @doc """
+  Return max buffer size of the low level SPI interface
+
+  This may be helpful when deciding transfer size.
+  """
+  @spec max_buf_size() :: integer() | :unknown
+  defdelegate max_buf_size(), to: Nif
```

Comment:
```suggestion
  @spec max_transfer_size() :: integer()
  defdelegate max_transfer_size(), to: Nif
```

In addition to the rename, I removed `:unknown`. I think we can spare users having to handle `:unknown` by giving them a minimum supported size.

Reply:
I agree with you. I was thinking about this, but I didn't know the minimal safe size. But it's definitely better to have some default value instead of doing more handling.